combinations/hash.tsv

```diff
@@ -392,4 +392,4 @@ bioconductor-clusterprofiler=4.6.0,bioconductor-org.hs.eg.db=3.16.0
 xopen=1.1.0,pysam=0.16.0
 bioconductor-purecn=2.4.0,bioconductor-txdb.hsapiens.ucsc.hg38.knowngene=3.16.0,bioconductor-org.hs.eg.db=3.16.0,bioconductor-txdb.hsapiens.ucsc.hg19.knowngene=3.2.2
 python=3.6.13,pandas=1.1.5,numpy=1.19.5,jinja2==2.11.3,pyyaml=5.3,fire=0.3
-python=3.7,pysam=0.16.0,flask=2.2.2,numpy=1.21.6,matplotlib=3.2.2,scipy=1.7.3,mosek::mosek=8.1.83,intervaltree=3.1.0,future=0.18.2
+python=3.7,pysam=0.16.0,flask=2.2.2,numpy=1.21.6,matplotlib=3.2.2,scipy=1.7.3,mosek::mosek=8.1.83,anaconda::intervaltree=3.1.0,future=0.18.2
```

Comment:
Ah now I see. Mosek is not coming from conda-forge or bioconda. That is probably the problem. Any chance to move this to conda-forge or bioconda?

Reply:
Sadly no. I asked the developers a few months ago and they are not eager to put it onto conda-forge or bioconda. This is an issue that always comes up and mosek 8 is required for the software. Newer versions of mosek do not work

Reply:
We don't need permission. It would be nice if they would maintain it in conda-forge but it is not needed, you can do that or anyone else.

Reply:
I tried to do it myself with the mosek pip version, but there only version 9 is available. I am unsure how I can do it with the older versions which I only find on their website or their mosek repository

Reply:
And you need version 8.1? 9 is not working? 
We should create a feedstock with version 9 and get the ball rolling, then we can use the feedstock to try to get an older version deployed.

Reply:
No, the software requires version 8. I am not the developer of PrepareAA, I am just maintaining a nf-core pipeline that uses the software. Before that I found a workaround, but now it is not possible anymore.

I am not sure what you mean with create a feedstock with version 9

Reply:
Would you be willing to merge ?

Reply:
@bgruening Did you decide on what to do with the pull request? I am currently putting the nf-core/circdna pipeline together for a new version release and this is the last hurdle to get it through the testing phase.

Reply:
I'm tempted to merge. But we can not include arbitrary packages from random channels. There is now guarantee of quality and this can quickly become a security nightmare. I'm afraid we need to wait until this package is available conda-forge :(

Reply:
The same mosek version is already in two of the multi-package containers, which were merged around a year ago. The only thing that is different to the container is intervaltree, which needed to be added for the new software version. I don't understand why this time it is different. 
```
python=2.7,pysam=0.15.2,flask=1.1.2,cython=0.29.14,numpy=1.16.6,scipy=1.2.1,matplotlib=2.2.5,mosek::mosek=8.0.60,future=0.18.2
python=2.7,pysam=0.17.0,flask=1.1.2,cython=0.29.14,numpy=1.16.6,scipy=1.2.1,matplotlib=2.2.5,mosek::mosek=8.0.60
```

Reply:
@bgruening I now upgraded to python3 and mosek9 which seems to work with the software. I will let you know if this will break something based on comments from the software maintainers. Would this be an adequate solution for you to consider merging ?

Reply:
@bgruening I had a discussion with the software maintainer and it is indeed usable with mosek9, which fixes all dependencies on python2 (https://github.com/nf-core/circdna/issues/52) . I also tested it in the nf-core pipeline and the multi container seems to be working.

Reply:
@bgruening is anything else needed?